README.md

```diff
@@ -113,7 +113,7 @@ If you are interested in JavaScript, TypeScript, React, Redux, or Design Systems
 ## Contributing
 
 This project welcomes contributions and suggestions. Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
```

Comment:
The changes "Contributing" section needs to be reverted - these are standard text from our legal.